keyboards/rura66/rev1/info.json

```
@@ -0,0 +1,18 @@
+{
+  "keyboard_name": "RURA66",
+  "url": "https://github.com/ShinoharaTa/keyboards/Rura66",
+  "maintainer": "T.Shinohara",
```

This is meant to reference your GitHub username.

```suggestion
  "maintainer": "ShinoharaTa",
```